packages/dd-trace/src/plugins/util/web.js

```diff
@@ -429,13 +496,13 @@ function addRequestTags (context) {
   const { req, span, config } = context
   const url = extractURL(req)
 
-  span.addTags({
+  span.addTags(Object.assign({
     [HTTP_URL]: web.obfuscateQs(config, url),
     [HTTP_METHOD]: req.method,
     [SPAN_KIND]: SERVER,
     [SPAN_TYPE]: WEB,
     [HTTP_USERAGENT]: req.headers['user-agent']
-  })
+  }, web.extractIp(context)))
```

Comment:
Similar to the above, always setting the tags would remove the need for the assign.

Reply:
Same anser as above